src/electos/ballotmaker/data/edf-to-ballot-data.py

```diff
@@ -0,0 +1,43 @@
+import argparse
+import json
+from dataclasses import asdict
+from pathlib import Path
+
+from electos.ballotmaker.data.extractor import BallotDataExtractor
+
+
+def report(data, **opts):
+    """Generate data needed by BallotLab."""
+    extractor = BallotDataExtractor()
+    ballot_data = extractor.extract(data)
+    ballot_data = [asdict(_) for _ in ballot_data]
+    print(json.dumps(ballot_data, indent = 4))
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "file", type = Path,
+        help = "Test case data (JSON)"
+    )
+    parser.add_argument(
+        "--debug", action = "store_true",
+        help = "Enable debugging output and stack traces"
+    )
+    opts = parser.parse_args()
+    file = opts.file
+    opts = vars(opts)
+
+    try:
```

Comment:
Sure. But there's not much to it, we can just copy it out. It's not a separate loader because I don't see the script as being used except to test out results. I presume the rest of BallotMaker will be managing files and passing them in. You'll note that `BallotDate.Extractor.extract` takes a dictionary, not a file or a stream.

I'm trying this out now on a branch: modifying `ballotmaker demo` so it takes a `Path` parameter and loads the test case from that parameter.

Reply:
Cool! Let me know how it works out